.vscode/launch.json

```diff
@@ -0,0 +1,21 @@
+{
+    // Use IntelliSense to learn about possible attributes.
```

Comment:
1: These look like vscode config files. Should they be gitignore-d?
2: I'll assume the comments are valid since they look auto-gen'd, but json comments are unfortunately not spec compliant

Reply:
I was planning on keeping them in the repository if that's okay. Creating them is a pain because I never remember how to do it, and when I reset a VM and lose them, I have to start from scratch.

Reply:
No worries. This is the kind of thing that by default I wouldn't want included. That's a good reason to make an exception though, as long as nothing secret makes its way into the repo.